ApplySyntax.sublime-settings

```diff
@@ -181,7 +181,7 @@
         },
         {
             "syntax": "JavaScript/JSON",
-            "extensions": [".jshintrc", ".jscsrc", ".csslintrc", ".eslintrc", ".bowerrc"]
+            "extensions": [".jshintrc", ".jscsrc", ".csslintrc", ".eslintrc", ".bowerrc", "geojson"]
```

Comment:
Thanks, I was just making sure.  I don't know how geojson files normally look, so I just wanted to ask before I merge.